packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -40,62 +47,73 @@ interface UserHeaderItemProps {
+            startIcon={isMobile ? undefined : <UserHeaderAccountIcon impersonated={user.impersonated} />}
+        >
+            <DropdownContent padding={0}>
+                <Box padding={4}>
+                    <Typography color="textSecondary" variant="caption" sx={{ paddingBottom: 2, display: "block" }}>
+                        <FormattedMessage id="comet.logged.in" defaultMessage="Logged in as" />
+                    </Typography>
+                    <Typography variant="h4">{user.authenticatedUser ? user.authenticatedUser.name : user.name}</Typography>
+                    <Typography variant="body2">{user.authenticatedUser ? user.authenticatedUser.email : user.email}</Typography>
+                </Box>
+
                 <Separator />
-                {user.impersonated && (
+
+                {user.impersonated && <ImpersonationInlay />}
+                {children && (
                     <>
-                        <StopImpersonationButton
-                            startIcon={<Clear />}
-                            fullWidth
-                            variant="outlined"
-                            color="primary"
-                            sx={{ justifyContent: "center" }}
-                        />
+                        <Box padding={4}>{children}</Box>
                         <Separator />
                     </>
                 )}
-                {isSigningOut ? (
-                    <Loading />
-                ) : (
-                    <Button
-                        fullWidth
-                        variant="contained"
-                        color="primary"
-                        startIcon={<Logout />}
-                        onClick={async () => {
-                            const result = await signOut();
-                            if (result.data) {
-                                location.href = result.data.currentUserSignOut;
-                            }
-                        }}
-                        sx={{ justifyContent: "center" }}
-                    >
-                        <FormattedMessage id="comet.logout" defaultMessage="Logout" />
-                    </Button>
-                )}
+                <Box padding={4}>
+                    {isSigningOut ? (
+                        <Loading />
+                    ) : (
+                        <Button
+                            fullWidth
+                            variant="contained"
+                            color="primary"
+                            startIcon={<Logout />}
+                            onClick={async () => {
+                                const result = await signOut();
+                                if (result.data) {
+                                    location.href = result.data.currentUserSignOut;
+                                }
+                            }}
+                            sx={{ justifyContent: "center" }}
```

Comment:
Custom button styling should not be needed

Reply:
Without this the button text won t be centered

Reply:
The default styling should be sufficient. 
Is this the issue? Can this be removed or just not used here? 

https://github.com/vivid-planet/comet/blob/bf8905bd4004e90c0643ce3d865982be1d31ab5e/packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx#L20-L22